ansible/roles/migrate-kubo-c1/tasks/main.yml

```diff
@@ -0,0 +1,135 @@
+---
+# Script to update the ceramic-one blockstore and run migration on the updated blocks
+
+# Basic plan:
+
+# Get latest common snapshot
+
+# create new snapshot and send incremental update since latest common snapshot from kubo -> c1
+
+# get file list of changed files and dates between dates of latest common snapshot & new snapshot
+
+# diff with processed list of files and dates
+
+# run migration on the todos (not yet processed) and add to processed list as each completes
+
+##################################################################################################
+######   Send Incremental Snapshot ###############################################################
+
+# Get latest common snapshot between the kubo and c1 datastores
+- name: Get latest common snapshot between gitcoin-go-ipfs-1 and gitcoin-rust-ceramic-1
+  block:
+    - name: List snapshots from gitcoin-go-ipfs-1
+      ansible.builtin.shell:
+        cmd: zfs list -H -t snapshot -o name ipfspool/data-store
+      register: kubo_snapshots
+      delegate_to: gitcoin-go-ipfs-1
+
+    - name: Let snapshots from gitcoin-rust-ceramic-1
+      ansible.builtin.shell:
+        cmd: zfs list -H -t snapshot -o name migrationpool/data-store
+      register: c1_snapshots
+      delegate_to: gitcoin-rust-ceramic-1
+
+    - name: Find latest common snapshot
+      ansible.builtin.shell:
+        cmd: |
+          kubo_snaps="{{ kubo_snapshots.stdout_lines | join('\n') }}"
+          c1_snaps="{{ c1_snapshots.stdout_lines | join('\n') }}"
+          echo "$kubo_snaps" | grep -F "$(echo "$c1_snaps" | sed 's/migrationpool\/data-store@//')" | tail -n 1
+      register: common_snapshot
+      failed_when: common_snapshot.rc != 0 or common_snapshot.stdout == ""
+      delegate_to: localhost
```

Comment:
no, i don't think so, i think we just want to bring over everything since the latest common 

if we make holes we can fix them manually, we want this whole process done in the next day or so